apps/users/handlers.py

```diff
@@ -155,13 +212,34 @@ async def delete_user(university_id: int, delete_user: DeleteUserIn, auth=Depend
         "data": {
             "user_id": delete_user.user_id
         },
-        "message": f"Deleted user with id {delete_user.user_id}"
+        "message": f"Deleted user with id {delete_user.user_id}",
+        "code": http_status.HTTP_202_ACCEPTED
```

Comment:
For a DELETE request: HTTP 200 or HTTP 204 should imply "resource deleted successfully".

HTTP 202 can also be returned by either operation and would imply that the instruction was accepted by the server, but not fully applied yet.

Reply:
Ok. What code should I use?

Reply:
204